packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMediaGroup.ts

```diff
@@ -144,6 +150,79 @@ describe('RemoteMediaGroup', () => {
 
   });
 
+  describe('setNamedMediaGroup', () => {
+    it('updates named media group', () => {
+
+      const nameGroup1 = { type: 1, value: 20 };
+      const nameGroup2 = { type: 1, value: 24 };
+      const group = new RemoteMediaGroup(fakeMediaRequestManager, fakeNamedMediaSlots, 255, true, {
+        namedMediaGroup: nameGroup1,
+      });
+      fakeMediaRequestManager.addRequest.resetHistory();
+      group.setNamedMediaGroup(nameGroup2, false);
+
+      assert.calledOnce(fakeMediaRequestManager.cancelRequest);
+
+      assert.calledOnce(fakeMediaRequestManager.addRequest);
+
+      assert.calledWith(
+        fakeMediaRequestManager.addRequest,
+        sinon.match({
+          policyInfo: sinon.match({
+            policy: 'active-speaker',
+            priority: 255,
+            nameMediaGroups: sinon.match.any,
```

Comment:
we should check the corrrect values here, not "any"

Reply:
Fixed